README.md

```diff
@@ -518,3 +518,61 @@ assets:
 | ------ | -------------------------- | --------------- |
 | `@doc` | `Nokogiri:: XML::Document` | `image/svg+xml` |
 | `@doc` | `Nokogiri::HTML::Document` | `image/*`       |
+
+## Migrating from Earlier Versions
+
+### Configuration
+
+Before
+
+```yaml
+cdn: https://example.com
+```
+
+After
+
+```yaml
+cdn:
+  url: https://example.com
+```
+
+### Images/CSS/JS
+
+Before
+
+```liquid
+{% img image.jpg width:"60" class:"image" %}
+```
+
+After
+
+```liquid
+{% asset image.jpg width="60" class="image" %}
+```
+
+Before
+
+```liquid
+{% css css.css %}
+```
+
+After
+
+```liquid
+{% asset css.css %}
+```
```

Comment:
You can probably merge these one to one into the previous "before" and "after" because they are related really. I like to be efficient when explaining things.